### PR TITLE
Update references of Extension Workshop to Firefox Extension Workshop

### LINKED ISSUE #593: Update references of Extension Workshop to Firefox Extension Workshop
Related to #592. To improve SEO, we are updating the name of this site to Firefox Extension Workshop. 

For this issue, we need to: 

- Find all instances in the content of this site where we refer to it as Extension Workshop 
- Update those instances to be Firefox Extension Workshop

### PR DESCRIPTION
**Pull request by AyahRamahi**
Fixes #593

### CODE REVIEW

File: _includes/guides-header.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h3 class="logo">
 					</ul>
 
 					<p><a href="https://addons.mozilla.org/developers/addons" class="button" target="_blank" rel="noreferrer noopener">Edit Product Page</a></p>
-					<p><a href="{{ "/" | relative_url }}" class="back">Extension Workshop</a></p>
+					<p><a href="{{ "/" | relative_url }}" class="back">Firefox Extension Workshop</a></p>
```
Comment (caitmuenster) [inline _includes/guides-header.html]: After previewing this on a local build, I think we can keep the header "Extension Workshop" and let the logo communicate the branding. :) 

Can you revert this?

File: _includes/header.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <header class="site-header">
     <div class="grid-x grid-padding-x">
         <div class="cell small-12">
-            <h3 class="logo"><a href="{{ "/" | relative_url }}">Extension Workshop</a></h3>
+            <h3 class="logo"><a href="{{ "/" | relative_url }}">Firefox Extension Workshop</a></h3>
```
Comment (caitmuenster) [inline _includes/header.html]: @muffinresearch Should this be reverted as well if we're not changing the header?

**Comment by muffinresearch** [comment]
@caitmuenster Leaving the Firefox out for now is a good call. Ideally we should have a text alternative for the logo so that the heading is actually "Firefox Extension Workshop" in text even if the visual representation is `[LOGO] Extension Workshop`. I'll file a separate issue for that.

**Comment by caitmuenster** [comment]
Woohoo, thanks @AyahRamahi! 🎉 Your contribution has been added to our [recognition wiki](tps://wiki.mozilla.org/Add-ons/Contribute/Recognition). 

Would you be interested in creating an account on mozillians.org? I'd be happy for vouch for you!

**Comment by AyahRamahi** [comment]
@caitmuenster I already have an account, and it would be amazing to be vouched!

**Comment by caitmuenster** [comment]
Awesome! You're vouched, @AyahRamahi! Welcome onboard! We look forward to seeing you around.

**Pull request merged**